clouddriver-eureka/src/main/groovy/com/netflix/spinnaker/clouddriver/eureka/deploy/ops/AbstractEurekaSupport.groovy

```diff
@@ -336,8 +351,8 @@ abstract class AbstractEurekaSupport {
   }
 
   enum DiscoveryStatus {
-    Enable('UP'),
-    Disable('OUT_OF_SERVICE')
+    UP('UP'),
+    DOWN('OUT_OF_SERVICE')
```

Comment:
`DiscoveryStatus.UP` makes sense, but `DiscoveryStatus.DOWN` not so much. DOWN and OUT_OF_SERVICE are 2 different Discovery statuses, DOWN means unhealthy (red) whereas OUT_OF_SERVICE means healthy but disabled (gray).

If it's because OUT_OF_SERVICE is too long, `DiscoveryStatus.OOS` might be ok?

Reply:
Oh, thanks for clarifying. I'll rename to `OUT_OF_SERVICE`.